src/main/java/frc/robot/subsystems/swerve/SwerveModule.java

```diff
@@ -1,267 +1,272 @@
-    steerMotor.clearStickyFaults();
-
-    // Get the initial position and velocity of the encoders
-    driveEncoderPosition = driveMotor.getSelectedSensorPosition();
-    steerEncoderPosition = steerMotor.getSelectedSensorPosition();
-    driveEncoderVelocity = driveMotor.getSelectedSensorVelocity();
-    steerEncoderVelocity = steerMotor.getSelectedSensorVelocity();
-
-
-    // Set the encoder coefficients for the drive and steer motors
-    driveMotor.configSelectedFeedbackCoefficient(ModuleConstants.kDriveEncoderRot2Meter);
-    driveMotor.configSelectedFeedbackCoefficient(ModuleConstants.kDriveEncoderRPM2MeterPerSec);
-    steerMotor.configSelectedFeedbackCoefficient(ModuleConstants.kSteerEncoderRot2Rad);
-    steerMotor.configSelectedFeedbackCoefficient(ModuleConstants.kSteerEncoderRPM2RadPerSec);
-
-    // Initialize the drive and steer PID controllers using the constants from DriveConstants
-    drivePIDController = new PIDController(DriveConstants.kPSwerveDriveDriveMotor, DriveConstants.kISwerveDriveDriveMotor, DriveConstants.kDSwerveDriveDriveMotor);
-    steerPIDController = new PIDController(DriveConstants.kPSwerveDriveSteerMotor, DriveConstants.kISwerveDriveSteerMotor, DriveConstants.kDSwerveDriveSteerMotor);
-    steerPIDController.enableContinuousInput(-Math.PI, Math.PI);
-
-    // Reset the encoder positions to zero
-    driveMotor.setSelectedSensorPosition(0); // Resets the  Drive Motor Encoder
-    steerMotor.setSelectedSensorPosition(0); // Resets the Steer Motor Encoder
-
-    // Setting Integrator Range (I in PID) | (Makes sure we don't go over the voltage limit)
-    drivePIDController.setIntegratorRange(-ModuleConstants.kFalcon500Voltage, ModuleConstants.kFalcon500Voltage);
-  }
-
-  //Accessor Methods for private instance variables
-  public double getDriveEncoderPosition() {
-    return driveEncoderPosition;
-  }
-
-  public double getSteerEncoderPosition() {
-    return steerEncoderPosition;
-  }
-
-  public double getDriveEncoderVelocity() {
-    return driveEncoderVelocity;
-  }
-
-  public double getSteerEncoderVelocity() {
-    return steerEncoderVelocity;
-  }
-
-  public boolean getAbsoluteEncoderReversed() {
-    return absoluteEncoderReversed;
-  }
-
-  public double getTurningMotorOutput() {
-    return turningMotorOutput;
-  }
-
-  public boolean getIsDriveMotorReversed() {
-    return isDriveMotorReversed;
-  }
-
-  public boolean getIsSteerMotorReversed() {
-    return isSteerMotorReversed;
-  }
-
-
-  private void saveEncoderOffset() {
-    try {
-      BufferedWriter writer = new BufferedWriter(new FileWriter(FilePathConstants.steerEncoderOffsetSavesPath));
-      for (int i = 0; i < 4; i++) {
-        writer.write(Double.toString(this.absoluteEncoderOffsetRad));
-        writer.newLine();
-      }
-      writer.close();
-    } catch (IOException e) {
-      System.out.println("\u001b[31;1mFailed to write turn encoder offsets to file.\u001b[0m");
+    private static double[] turnEncoderOffsets;
```

Comment:
Fix the formatting on this file so the diff isn't so large, there's no reason to change the indentation.